docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,227 @@
+# Rally Release Process
+The Rally Core Add-on requires special Firefox-only privileges and so it must use the Mozilla signing infrastructure.
+This requires Rally's main GitHub repository (`github.com/mozilla-ion/ion-core-addon`) to be cloned to the `github.com/mozilla-extensions` organization, and have a working `.taskcluster.yml` file in the repository. It is important to note that this must be a clone and not a fork, otherwise the signing infrastructure will not work.
```

Comment:
I don't understand this paragraph at all.
Is this something that needs to re-happen for every release or will there be a repository under mozilla-extensions that gets pushed to?
The differentation between "clone" and "fork" is not clear here, I guess a "fork" in the sense of GitHub where it explicitly marks it as forked from this repo?

At best you want to start this document with at least one sentence explaining what the content will be.
Something a la "Rally will be shipped as a web extensions, with special Firefox-only privileges. [...] In order to enable Firefox-specific privileges it requires a signature done on Mozilla infrastructure"

Reply:
> Is this something that needs to re-happen for every release or will there be a repository under mozilla-extensions that gets pushed to?

Thanks for going through this! There is a repository under [`mozilla-extensions`](https://github.com/mozilla-extensions/ion-core-addon/) against which we need to open PRs to have the addons signed.

> The differentation between "clone" and "fork" is not clear here, I guess a "fork" in the sense of GitHub where it explicitly marks it as forked from this repo?

Yes, "fork" in the sense of GitHub. I will link to the [GitHub Glossary](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/github-glossary#fork) to make it clearer.

> At best you want to start this document with at least one sentence explaining what the content will be.
> Something a la "Rally will be shipped as a web extensions, with special Firefox-only privileges. [...] In order to enable Firefox-specific privileges it requires a signature done on Mozilla infrastructure"

Will do!